src/main/target/CC3D/target.h

```diff
@@ -44,7 +44,7 @@
 
 #define USABLE_TIMER_CHANNEL_COUNT 12
 
-#define DEBUG_MPU_DATA_READY_INTERRUPT
+//#define DEBUG_MPU_DATA_READY_INTERRUPT
```

Comment:
oh, gotcha, it's a debug config flag.

Disregard then (but you're right that it might need to be commented on master)